packages/client/src/clients/guide/client.ts

```diff
@@ -62,6 +63,18 @@ const checkForWindow = () => {
 export const guidesApiRootPath = (userId: string | undefined | null) =>
   `/v1/users/${userId}/guides`;
 
+// Detect debug params like "knock_guide_key" from URL.
+const detectDebugParams = (): DebugState => {
+  if (typeof window === "undefined") {
+    return { forcedGuideKey: null };
+  }
+
+  const urlParams = new URLSearchParams(window.location.search);
+  const forcedGuideKey = urlParams.get("knock_guide_key");
```

Comment:
Eventually we will want to pull out all the supported debug query params and set as constants.